templates/snippets/breadcrumbs.html

```diff
@@ -0,0 +1,15 @@
+{% load static %}
+<nav class="breadcrumbs">
+    <ol>
+        {% for crumb in breadcrumbs %}
```

Comment:
It would be more readable to use `name, url` (or similar) instead of `crumb.0` and `crumb.1` here